hack/install-e2e-dependencies.sh

```diff
@@ -0,0 +1,25 @@
+set -eux
+curl -Lo helm.tar.gz \
+     https://storage.googleapis.com/kubernetes-helm/helm-v2.6.1-linux-amd64.tar.gz
+tar xvf helm.tar.gz
+sudo mv linux-amd64/helm /usr/local/bin
+
+curl -Lo kubectl \
+     https://storage.googleapis.com/kubernetes-release/release/$KUBERNETES_VERSION/bin/linux/amd64/kubectl
+chmod +x kubectl
+sudo mv kubectl /usr/local/bin/
+
+curl -Lo minikube \
+     https://storage.googleapis.com/minikube/releases/v0.23.0/minikube-linux-amd64
+chmod +x minikube
+sudo mv minikube /usr/local/bin/
```

Comment:
Doesn't need to be done for this, as it's a whole new unit of feature, but now that we're doing this in a script in hack (i.e. it's more likely to be run by developers and not just CI), I think we shouldn't make changes outside of the repo root/in `/usr/local/bin`